src/components/button/index.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import classes from './styles.module.css';
+import SnackBar from '../snackbar/index';
+import {isEmptyString} from '../utils/index';
+
+const Button = props =>{
+    const {validity,inputTextVal,handleInputTextChange} = props;
+    const [openSnackbar,setOpenSnackbar]=useState(false);
+    const [snackbarMessage,setSnackbarMessage]=useState('');
+    const [snackbarSeverity,setSnackbarSeverity]=useState('');
+
+    const handleButtonClick=()=>{
```

Comment:
wrap with useCallback

Reply:
Done!.......but need to discuss  the use over here.